website/quantum-computing-primer/01_introduction/index.md

```diff
@@ -70,16 +76,19 @@ qc.apply_hadamard_gate(0)
   
 # Execute the circuit and measure the result  
 result = qc.execute_circuit()  
-print(result)  
-```
+print(result)   
+``` -->
 
 In this example, we:
 
 * Created a quantum circuit with 1 qubit.
 * Applied a Hadamard gate to put the qubit into a superposition state.
 * Measured the qubit to observe the probabilistic outcome.
 
-This is just the beginning! In the next sections, you'll dive deeper into quantum gates, circuits, and algorithms using qumat.
+# proposed method, does not yet exist
```

Comment:
? not sure what is going on here. 

What I meant was in the later examples, let's say to actually do the example we needed some currently unimplemented gate, let's call it `foo_gate`. then in the example, you can re write it to be correct, but call out in the code that `foo_gate` doesn't exist yet.